src/stylesheets/elements/_buttons.scss

```diff
@@ -193,3 +193,140 @@
     font-size: 14px;
   }
 }
+//SDS Statistics
```

Comment:
can I apply statistics style to a link(a tag)? what about if I just want to use it to style a text? The following code indicates that this can only be used as a button. Maybe it will be better to move this to its own page similar to this: https://semantic-ui.com/views/statistic.html